msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,205 @@
+---
+title: Preapproval of RSC permissions
+description: In this article, learn more about to create, manage, and delete preapprovals of RSC enabled app in Microsoft Teams.
+ms.localizationpriority: medium
+author: MSFTRickyCastaneda
+ms.author: rcastaneda
+ms.topic: conceptual
+ms.date: 08/29/2023
+---
+
+# Preapproval of RSC permissions
+
+Resource-Specific Consent (RSC) permission preapproval provides admins with detailed control over the RSC permissions that an app can request during installation. The app only receives RSC permissions at the installation time. However, using preapproval policies, admins can predetermine the maximum permissions that an app can request from the end user and secure consent during installation.
+
+To successfully create and use a preapproval policy for an app, admins must ensure the app can be installed in their organization. For more information, see [managing apps in Teams Admin Center](/microsoftteams/apps-in-teams). Preapproval policies don't affect apps blocked through Teams Admin Center.
+
+Admins can create a detailed preapproval policy, based on the app ID, permissions, and the sensitivity of the accessed data. Preapproval of RSC permissions are designed for admins seeking to create advanced custom policies for their organization.
+
+By default, the preapproval policies are managed by Microsoft, and we recommend that organizations maintain `ManagedByMicrosoft` state. This state allows Microsoft's security team to deliver the best security for your organization.
+
+> [!NOTE]
+>
+> * Preapproval of RSC permissions are available only in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * Preapproval of RSC permissions and its operating procedures are subject to change.
+
+## Set up PowerShell to manage preapproval of RSC permissions
+
+Preapproval of RSC permissions are managed through Microsoft Graph PowerShell. You can learn more about managing Microsoft Teams with PowerShell [here](/powershell/microsoftgraph/get-started).
+
+Before you can create and manage preapproval policies, use `Connect-MgGraph` cmdlet to connect PowerShell to your tenant using Microsoft Graph. To create, manage, and delete RSC preapproval policies, add the following permissions:
```

Comment:
To create, manage, and delete RSC preapproval policies, you must grant the following permissions to the PowerShell cmdlet:

Reply:
Updated as suggested